src/@chakra-ui/gatsby-plugin/components/Badge.ts

```diff
@@ -0,0 +1,34 @@
+import { ComponentStyleConfig, SystemStyleObject } from "@chakra-ui/react"
+
+const variantSecondary: SystemStyleObject = {
+  borderColor: "primary100",
+  color: "text",
+}
+
+export const Badge: ComponentStyleConfig = {
+  baseStyle: {
+    borderRadius: "base",
+    border: "1px solid",
+    borderColor: "transparent",
+    color: "black300",
+    fontWeight: "initial",
+    py: 1,
+    px: 2,
+    textTransform: "uppercase",
+  },
```

Comment:
To be able to consolidate styles for instances that include a background color, it was a choice between either ensuring that all instances also included the "primary" border color (which all did not), or not include it at all. I chose not to include it here.

Having the minor border in all instances is not showing to impact the layouts.